### DISCUSSION TITLE
New Transparency Blending Mode: Multiply

### DISCUSSION THREAD

**Discussion by Kokorz** [ikemen-engine/Ikemen-GO#1654] [Feature Request]
### Is your feature request related to a problem? Please describe.

I'm currently working on a custom VS Screen for my game. A mock-up of it has already been completed, what I have to worry about is the implementation of the mockup.

To cut down on a vast majority of what could potentially be repeat assets, or just to make some stuff inside it cooler, I need to utilize different blending modes, particularly the ones that come with Adobe products. This engine, however, only has Additive and Subtractive blending, which while they are good, it isn't enough for my intentions.

### Describe the solution you'd like

Ultimately, I would love every blending mode that exists in Photoshop to be usable in Ikemen. Lighten, Screen, Multiply, Soft/Hard Light, Invert (I know the PalFX parameter exists, give it to everywhere else)

...But, that's in a perfect world. Because that likely isn't viable, I suggest adding the most essential of them, which will allow a lot of them to be replicated: Multiply.

It should be added to basically every place that allows you to specify blending modes (stage background controllers, animations/actions, char trans state controller, so on and so forth)

### Search terms

blend trans multiply

**Comment by SuperFromND** [comment]
I'm in favor of this. I use Multiply blending all the time in Photoshop and it wouldn't be *that* hard to implement outside of just adding it to all the relevant areas in the codebase; the blending algorithim is right there in the name!
